### PR TITLE
accounts rebranding

### PR DESCRIPTION
**Pull request by CatalinFaurPaul**

### CODE REVIEW

File: accounts/erd195fe57d7fm5h33585sc7wl8trqhrmy85z3dg6f6mqd0724ymljxq3zjemc.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Elrond Foundation",
+  "name": "Multiversx Foundation",
```
Comment (dragos-rebegea) [inline accounts/erd195fe57d7fm5h33585sc7wl8trqhrmy85z3dg6f6mqd0724ymljxq3zjemc.json:2]:
```suggestion
  "name": "MultiversX Foundation",
```

**Review by dragos-rebegea — CHANGES_REQUESTED** [review]

**Review by ioandanpopa — DISMISSED** [review]

File: accounts/erd1qqqqqqqqqqqqqpgqdt9aady5jp7av97m7rqxh6e5ywyqsplz2jps5mw02n.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "Maiar Exchange: Governance Module",
+  "name": "xExchange: Governance Module",
   "description": "",
   "tags": [
-    "mex",
+    "xexchange",
```
Comment (SebastianMarian) [inline accounts/erd1qqqqqqqqqqqqqpgqdt9aady5jp7av97m7rqxh6e5ywyqsplz2jps5mw02n.json:5]: It is ok this change?

File: accounts/erd195fe57d7fm5h33585sc7wl8trqhrmy85z3dg6f6mqd0724ymljxq3zjemc.json (hunk 1 above)
Comment (CatalinFaurPaul) [reply accounts/erd195fe57d7fm5h33585sc7wl8trqhrmy85z3dg6f6mqd0724ymljxq3zjemc.json]: Fixed

**Review by dragos-rebegea — DISMISSED** [review]

File: accounts/erd16rp9ur5crj6lcjyttr0ftft8vspmcgq3kk00wmzkv6p7lnqg3v8quhqhh5.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "Elrond Foundation: Protocol Treasury",
+  "name": "Multiversx Foundation: Protocol Treasury",
```
Comment (SebastianMarian) [inline accounts/erd16rp9ur5crj6lcjyttr0ftft8vspmcgq3kk00wmzkv6p7lnqg3v8quhqhh5.json]: Where Multiversx starts with caps M in this file, should not be MultiversX with caps X too?

Comment (CatalinFaurPaul) [reply accounts/erd16rp9ur5crj6lcjyttr0ftft8vspmcgq3kk00wmzkv6p7lnqg3v8quhqhh5.json]: fixed

File: accounts/erd1qqqqqqqqqqqqqpgqdt9aady5jp7av97m7rqxh6e5ywyqsplz2jps5mw02n.json (hunk 2 above)
Comment (CatalinFaurPaul) [reply accounts/erd1qqqqqqqqqqqqqpgqdt9aady5jp7av97m7rqxh6e5ywyqsplz2jps5mw02n.json]: fixed. reverted to "mex"

**Review by SebastianMarian — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]

**Pull request merged**